htdocs/includes/odtphp/Segment.php

```diff
@@ -170,8 +170,16 @@ public function macroReplace($text)
     {
         global $langs;
 
-        $patterns=array( '__CURRENTDAY__','__CURRENTDAYTEXT__','__CURRENTMONTHSHORT__','__CURRENTMONTH__','__CURRENTYEAR__' );
-        $values=array( date('j'), $langs->trans(date('l')), $langs->trans(date('M')), $langs->trans(date('F')), date('Y') );
+        $nextMonth = strtotime('+1 month');
```

Comment:
using strtotime does not take in consideration setup of user and timezone.
Can you use instead dolibarr function like dol_get_next_month, dol_getdate, dol_mktime, ...